torch/optim/adamw.py

```diff
@@ -140,7 +179,7 @@ def _init_group(self, group, params_with_grad, grads, amsgrad, exp_avgs, exp_avg
             if len(state) == 0:
                 state["step"] = (
                     torch.zeros((1,), dtype=torch.float, device=p.device)
-                    if self.defaults["capturable"]
+                    if self.defaults["capturable"] or self.defaults["fused"]
```

Comment:
Doesn't have to be fixed here but, as reported by a user, this is not ok in the case where the user overrides the "capturable" or "fused" flags only for a subset of the groups. It will lead to weird errors.

Reply:
accessing it in https://github.com/pytorch/pytorch/pull/94149